src/runtime/renderscript_dev.ll

```diff
@@ -0,0 +1,29 @@
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
Actually, those directives are needed in renderscript_dev.ll file. I put them back in 09e6be0.

Reason is that Renderscript code generation is done via compile_to_src, on a different path from CodeGen_LLVM::compile(), so CodeGen_Renderscript_Dev::get_target_triple() and get_data_layout() are not really being used. The methods have to be defined, though, as they are declared as abstract in parent classes.